src/_core/utils/MapWrapper.js

```diff
@@ -16,6 +16,23 @@ export default class MapWrapper {
         this.mapDate = appConfig.DEFAULT_DATE;
     }
 
+    init(container, options) {
+        console.warn("Error in MapWrapper.init: Method not implemented in subclass");
+        return false;
+    }
+    initBools(container, options) {
```

Comment:
Bikeshedding on this a bit – what about "initFlags"?

Reply:
I'm fine with either. Is there ever a time when a `true`/`false` variable wouldn't be called a flag?

Reply:
Probably not? Eh either way is fine with me.

Reply:
one reason to keep it `bool` could be because not all flags end up in that method. The `initializationSuccess` flag is set after all the other init functions

Reply:
Ah true. Ok bool is fine then.